### DISCUSSION TITLE
injecting css into iframes - did i break it?

### DISCUSSION THREAD

**Discussion by dallinmoak** [ankit/stylebot#687] [Q&A] [answered]
Hi, 
I gotta say, this is an incredible project and i love it very much. Thanks to everyone who contributed.

I have two questions: 1. is injecting css into the contents of iframes supported still? 
2. am I doing something wrong when using it? 

for an example, I'm trying to give all h1 tags that are direct descendants of body tags a red border, so i put `body > h1{border: 3px solid red;}` into the stylebot editor. 
on the www3school site [editor tool](https://www.w3schools.com/html/tryit.asp?filename=tryhtml_basic_document), there exists an iframe containing a body tag with an h1 descendant. Is this the correct way to style stuff within that iframe?
![reference1](https://user-images.githubusercontent.com/32078384/152070179-dd018786-7d27-42b9-9903-4012c127c1a7.png)
![reference2](https://user-images.githubusercontent.com/32078384/152070260-295ec87f-fe8f-4f5d-acf8-2dbda3c18958.png)

**Comment by joshseltzer** [comment, marked as answer]
@dallinmoak I am not sure why it stopped working, but I have implemented a fix here that works with your example: https://github.com/ankit/stylebot/pull/736

**Comment by rthomasdesign** [comment]
I don't know if it's broken again, but I'm also trying to style content within an iframe and it won't work.  Chrome Version 116.0.5845.188

**Comment by fahri314** [comment]
Iframe css selectors still does not work.
